package.json

```diff
@@ -23,7 +23,7 @@
     "gulp-gjslint": "^0.1.3",
     "jasmine-spec-reporter": "^0.6.0",
     "jsdoc": "^3.3.0-alpha9",
-    "karma": "^0.12.23",
+    "karma": "^0.13.16",
```

Comment:
Just curious and not against bumping the version of this up, but was the change from 0.12 to 0.13 needed?